test/integration/maincluster_test.go

```diff
@@ -1167,32 +1164,13 @@ experimental:
       - arn:aws:elasticloadbalancing:eu-west-1:xxxxxxxxxxxx:targetgroup/manuallymanagedetg/xxxxxxxxxxxxxxxx
     securityGroupIds:
       - sg-12345678
-  dex:
+  oidc:
     enabled: true
-    url: "https://dex.example.com"
-    clientId: "example-app"
-    username: "email"
+    url: "https://accounts.google.com"
+    clientId: "kubernetes"
+    username: "sub"
```

Comment:
If we use google as an example then I think username claim is `email` so I think fine to keep the way it was. At least, we have it confirmed with `email` and it works.